curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -550,7 +552,17 @@ MdsClientImpl::RefreshSession(const std::vector<PartitionTxId> &txIds,
             FsQuotaChecker::GetInstance().UpdateQuotaCache(
                 response.fscapacity(), response.fsusedbytes());
         }
-
+        if (response.has_mdsaddrsoverride()) {
```

Comment:
it's not a good idea to put heavy logic in here, `MdsClientImpl` is just used for communicate with MDS

Reply:
ok